### PR TITLE
docs: add tips to use jwt helper with dynamic secret

### PR DESCRIPTION
**Pull request by sor4chi**
スクリーンショット 2023-12-24 20 34 06

### CODE REVIEW

File: middleware/builtin/jwt.md [hunk 1]
```diff
@@ -54,6 +54,24 @@ app.get('/auth/page', (c) => {
...
+```js
+app.use(
+  '/auth/*',
+  (c, next) => {
+    const jwtMiddleware = jwt({
+      secret: c.env.JWT_SECRET,
+    })
+    return jwtMiddleware(c, next)
+  })
+)
```
Comment (yusukebe) [inline middleware/builtin/jwt.md:70]: This line should be removed.

File: middleware/builtin/jwt.md [hunk 2]
```diff
@@ -54,6 +54,24 @@ app.get('/auth/page', (c) => {
 })
 ```
 
+::: tip
+
+`jwt()` is just a middleware function. If you want to use dynamic secret (eg: `c.env.JWT_SECRET`) as a secret, you can use it as follows:
```
Comment (yusukebe) [inline middleware/builtin/jwt.md:59]: The following is better.

> If you want to use an environment variable

**Review by yusukebe — COMMENTED** [review]
Hey @sor4chi 

I've left the comments. Please check them.

**Comment by sor4chi** [comment]
@yusukebe Thanks, I fixed :)

**Comment by yusukebe** [comment]
@sor4chi 

Okay! Merging!

**Pull request merged**